components/function-controller/internal/controllers/integration_test.go

```diff
@@ -0,0 +1,391 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"testing"
+	"time"
+
+	serverless "github.com/kyma-project/kyma/components/function-controller/pkg/apis/serverless/v1alpha1"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
+	. "github.com/onsi/gomega/types"
```

Comment:
I don't know if I'm a big fan of dot-importing those libs, I know it's a common thing to do, but then I lose ability to see what exactly we're importing, eg by typing `gomega.` and having IDE autocomplete that

Reply:
it was autogenerated but we can change this to explicit imports when we migrate the content of the test to ginkgo